### DISCUSSION TITLE
Cannot use Playwright in Nuxt 3 and Vite

### DISCUSSION THREAD

**Discussion by GabriMcNab** [nuxt/nuxt#17388] [Questions] [answered]
Hello everyone,

I'm trying to add [Playwright](https://playwright.dev/docs/intro) to a new Nuxt 3 project, but as soon as I start to write the first test, I get this warning in the console, and even the `dev` build breaks:

<img width="1504" alt="Screenshot 2022-06-24 at 15 09 45" src="https://user-images.githubusercontent.com/36234022/175542684-0659e574-459b-4297-9088-fa6717744eea.png">

To reproduce:
1. download [this repo](https://github.com/GabriMcNab/nuxt-3-test)
2. Install the dependencies and run `npm run dev`
3. You should see the warning in the console, also, if you try to navigate to http://localhost:3000 you'll see that nothing is rendered
4. Try to remove `pages/__test__/index.test.ts` and re-launch the dev command
5. Now you should be able to see the app in http://localhost:3000

**Comment by Flanker72** [comment, marked as answer]
Nuxt tries to resolve your test as a plain page, however it cannot render it as a Vue component.
You may solve the issue in different ways:
1. Move your tests to another folder, outside the pages (see [Testing example](https://v3.nuxtjs.org/examples/advanced/testing))
2. Exclude `*.test.ts` files from analysing using [ignore](https://v3.nuxtjs.org/api/configuration/nuxt.config#ignore) section of `nuxt.config.ts`.
